concourse/scripts/unit_tests_gporca.bash

```diff
@@ -14,7 +14,7 @@ function build_xerces
 
 function test_orca
 {
-    if [ -n "${SKIP_UNITTESTS}" ]; then
+    if [ "${SKIP_UNITTESTS}" = false ]; then
```

Comment:
False? Don't we want to skip unit tests of `SKIP_UNITTESTS` is true?

Reply:
Fixed